source/developers-guide/event-guide/index.md

```diff
@@ -0,0 +1,185 @@
+which are already explained in the [controllers article](https://developers.shopware.com/developers-guide/controller/). 
+`Controller events` are defined in the `Enlight_Controller_Action` class which is located in `engine/Library/Enlight/Controller/Action.php`.
+As you can see in that file the events are generated dynamically so you are able to register on every controller's `action`, 
+`preDispatch` or `postDispatchSecure`.
+
+### preDispatch
+The `preDispatch` event will be executed before the dispatch process of a controller starts processing. 
+
+### postDispatch
+The `postDispatch` is executed after the dispatch process of the controller is finished. It is recommended 
+to use the `postDispatchSecure` event because you can be sure that this is a valid request. This event will only fired
+if a template is given. For example ajax requests and exceptions will not trigger this event.
+
+### actionEvents
+`actionEvents` will be fired everytime an action will be called. For example if the `indexAction` in the `Account` controller
+was called the event `Enlight_Controller_Action_Frontend_Account_Index` will be fired.
+
+### Event names
+The event names of the `controller events` are generated dynamically. We will use in this list placeholders for the event
+names so you are able to build the event name you need by yourself.
+
+This list is also sorted according to the time in which the events are fired. 
+
+* `Enlight_Controller_Action_PreDispatch`
+* `Enlight_Controller_Action_PreDispatch_MODULE`
+* `Enlight_Controller_Action_PreDispatch_MODULE_CONTROLLER`
+* `Enlight_Controller_Action_MODULE_CONTROLLER_ACTION`
+* `Enlight_Controller_Action_PostDispatchSecure_MODULE_CONTROLLER`
+* `Enlight_Controller_Action_PostDispatchSecure_MODULE`
+* `Enlight_Controller_Action_PostDispatchSecure`
+* `Enlight_Controller_Action_PostDispatch_CONTROLLER`
+* `Enlight_Controller_Action_PostDispatch_MODULE`
+* `Enlight_Controller_Action_PostDispatch`
+
+
+
+## Further events
+`Further events` will be fired in Shopware for example when a new resource will be registered by using the dependency 
+injection service.
+
+* `Enlight_Bootstrap_InitResource_CLASS`
+* `Enlight_Bootstrap_AfterInitResource_CLASS`
+* `Enlight_Controller_Dispatcher_ControllerPath_MODULE_CONTROLLER`
+
+The `Enlight_Bootstrap_InitResource` events will be used to register new namespaces and components.
+`Enlight_Controller_Dispatcher_ControllerPath_MODULE_CONTROLLER` will be used to register Plugin controllers.
+
+## Application events
+`Application events` are events that are explicitly emitted in order to allow you certain modifications. In contrast to 
+the very generic global events, application events have certain use cases in mind. For example:
+* do not allow this customer to log in
+* do not allow this product to be put in the cart
+* modify the result of the current product query
+* invalidate the cache for a certain prodcut id
+* register a new command line tool
+
+### Notify events
```

Comment:
ALL events fall into one of these 3 categories, but your guide does not make this clear. Please fix this